.gitignore

```diff
@@ -21,6 +21,7 @@ build
 # other
 eclipse
 run
+src/main/generated/
```

Comment:
Why ignore this?

Reply:
Including this generated directory in the source code would be redundant as the data generation code itself is already included.